cypress/e2e/search_spec.cy.js

```diff
@@ -0,0 +1,17 @@
+beforeEach(() => {
+  cy.intercept('http://localhost:8080/api/v1/places', {
```

Comment:
Interesting to see the `beforeEach` and `intercept` outside of the describe block. I thought they had to be nested within it. I'm curious how everyone else has been writing it?